test/src/test/kotlin/graphql/nadel/tests/CentralSchemaTesting.kt

```diff
@@ -5,12 +5,13 @@ import com.fasterxml.jackson.module.kotlin.readValue
 import graphql.GraphQLError
 import graphql.language.AstPrinter
 import graphql.nadel.Nadel
-import graphql.nadel.NadelExecutionInput.newNadelExecutionInput
+import graphql.nadel.NadelExecutionInput.Companion.newNadelExecutionInput
```

Comment:
This is unfortunate for Java consumers - but so be it

Reply:
Ah I think that's just how Kotlin imports it. I've actually added `@JvmStatic` to the `NadelExecutionInput.newNadelExecutionInput` so that it'll remain the though. But yea some code will have some indirections due to Kotlin objects etc.